src/NServiceBus.Core/Features/FeatureConfigurationContext.cs

```diff
@@ -66,7 +80,7 @@ public void AddSatelliteReceiver(string name, string transportAddress, PushRunti
             Guard.AgainstNull(nameof(recoverabilityPolicy), recoverabilityPolicy);
             Guard.AgainstNull(nameof(onMessage), onMessage);
 
-            Receiving.AddSatelliteReceiver(name, transportAddress, runtimeSettings, recoverabilityPolicy, onMessage);
+            Receiving.AddSatelliteReceiver(name, transportAddress, runtimeSettings, recoverabilityPolicy, (b, d, ctx) => onMessage(b, ctx));
```

Comment:
Technically the closure is only needed because I went ahead and exposed the dispatcher directly